docker-compose.yml

```diff
@@ -27,6 +32,22 @@ services:
         - .env
       ports:
         - 8080:8080
+      links:
+        # Link with the DB container as `db.local`
+        - db:db.local
+      # TODO we want a `depends` here, but only for using the local db container. So need an override file
```

Comment:
add a dot. also does this work as it is now? or do we need to override in order for it to work?

Reply:
this works - i am saying we will need an override in order to use a `depends`, because e.g. the production environment doesn't actually depend on a db container